openmetadata-ui/src/main/resources/ui/src/pages/CustomizablePage/CustomizablePage.tsx

```diff
@@ -73,22 +89,46 @@ export const CustomizablePage = () => {
   const handleSave = async () => {
     try {
       const finalPage = getFinalLandingPage(editedPage);
+      let response: Document;
 
       if (page.id) {
         const jsonPatch = compare(page, finalPage);
 
-        await updateDocument(page?.id ?? '', jsonPatch);
+        response = await updateDocument(page.id ?? '', jsonPatch);
       } else {
-        await createDocument(finalPage);
+        response = await createDocument(finalPage);
       }
+      setPage(response);
+      setEditedPage(response);
```

Comment:
Why we need two state variable here?

Reply:
It is for the comparison for page data change. As the layout change handlers are in child components, I've kept separate states to store the edited layout to keep track of the layout change.